extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -530,7 +574,12 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 				return [];
 			}
 
-			let cmd = `"${pythonExePath ?? this.pythonExecutable}" -m pip list --format=json`;
+			let cmd = '';
```

Comment:
Could set this to the common part - `"${pythonExePath ?? this.pythonExecutable}" -m pip list --format=json` and then only append the --user if user is specified.